DemoApp/Sources/Components/Gleap/GleapAdapter.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamVideo
+#if canImport(Gleap)
+import Gleap
+#endif
+
+final class GleapAdapter {
+
+    var isAvailable: Bool {
+        #if canImport(Gleap)
+        return true
+        #else
+        return false
+        #endif
+    }
+
+    init() {
+        #if canImport(Gleap)
+        Gleap.initialize(withToken: "qsoyideVaZ7ZOehLLl8LqDC6YOPagLvr")
+        Gleap.showFeedbackButton(false)
+        log.debug("✅ Gleap has been activated.")
+        #else
+        log.warning("Cannot enable Gleap as the module hasn't been imported.")
+        #endif
+    }
+
+    func login(_ user: User) {
+        guard isAvailable else {
+            return
+        }
+        let userProperty = GleapUserProperty()
```

Comment:
Seems like you haven't pushed this file - this is missing.

Reply:
yeap they are missing for UIKit. I'm adding them now